TutorLizard.BusinessLogic/Data/IStudentDataAccess.cs

```diff
@@ -1,5 +1,12 @@
-﻿namespace TutorLizard.BusinessLogic.Data;
+﻿using TutorLizard.BusinessLogic.Models;
+
+namespace TutorLizard.BusinessLogic.Data;
 
 public interface IStudentDataAccess
 {
+    AdRequest CreateAdRequest(int adId, object value, bool isAccepted, string message);
```

Comment:
Change object value to match signature that already exists in `DataAccess`:
`public AdRequest CreateAdRequest (int adId, int studentId, bool isAccepted, string message)`